src/main/java/nextstep/courses/service/SessionService.java

```diff
@@ -25,9 +27,24 @@ public Session createSession(Session session){
 
     public void registerStudent(NsUser student, Long sessionId){
         Session session = sessionRepository.findById(sessionId);
-        Payment payment = paymentService.payment(session.getId(), student.getId());
+        Payment payment = paymentService.findPayment(session.getId(), student.getId());
 
         session.addStudent(student, payment);
         sessionRepository.saveStudents(session);
     }
+
+    public void acceptStudent(NsUser teacher, Long sessionId, SelectedStudents students){
+        checkTeacher(teacher);
```

Comment:
```suggestion
        teacher.승인으로상태변경(students);
        vs 
        students.승인으로상태변경(teacher);
```

도메인에게 상태 변화의 책임을 주는건 어떨까요?